supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+* Avoid directional indicators such as left, right, above and below when providing navigational instruction in the user interface.
+
+=== Well-formed HTML and Meaningful Sequence
+
+*Accessibility criteria:* The meaning of content relies on the order in which you present it. For example, English content is arranged from left to right and readers usually look at a left-hand column before a right-hand column. Users who rely on assistive technology (such as a screen reader) to interpret content need the content to be presented in a meaningful order. If content is presented out of sequence, readers may become disoriented and unable to understand the content.
+
+* Correctly nest headings from H1 to H2, H3 and so forth without skipping heading levels to indicate relative importance.
+* Use correct Asciidoc tags to produce intended HTML.
+
+
+.Reference
+
+*  Understanding WCAG 2.0
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html[Text Alternatives].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-text-presentation.html[Images of Text].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html[Link Purpose(In Context)].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-understanding.html[Sensory Characteristics].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html[Use of Color].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html[Meaningful Sequence]
+* link:https://webaim.org/resources/contrastchecker/[WebAIM Contrast Checker].
+
+== Localization
+
+Documentation that follows the IBM Style Guide for fairly conversational tone does not present issues with human- or machine-translation. Use of contractions, specifically, has not shown to produce issues with translations.
+
+[[microcopy]]
+== Microcopy
+
+The words in a user interface, commonly referred to as "UX copy" or "microcopy," are just as important as the components or layouts. UX copy is another element of design, and it can drive better UX decisions and guide users to succeed. Red Hat cloud services are based on Patternfly, an open source design system created to enable consistency and usability across a wide range of applications and use cases.
+
+See link:https://www.patternfly.org/v4/ux-writing/about[UX writing] in the Patternfly content style guide for comprehensive guidelines when writing for the user interface.
+
+[[screenshots]]
+== Screenshots
+Avoid screenshots for both accessibility and localization concerns, if you must use screenshots use them as judiciously as possible and ensure alt text is unique and descriptive. See xref:accessibility[Accessibility] for more information on proper use of images in user interface documentation.
```

Comment:
Suggest breaking the first sentence into two.

"Avoid screenshots for both accessibility and localization reasons. If you must use screenshots, use them as judiciously as possible and ensure that the alt text is unique and descriptive."

Reply:
@bergerhoffer completed